### PR TITLE
Extend contributor guide

### PR DESCRIPTION
**Pull request by fzimmermann89**
Small change to your documentation

### CODE REVIEW

File: docs/source/contributor_guide.rst [hunk 1]
```diff
@@ -65,17 +65,54 @@ src/mrpro structure
...
+
+Building the Documentation
+==========================
+You can build the documentation locally via running ```make html``` in the docs folder. The documentation will also be build in each PR and can be viewed online.
+Please check how your new additions render in the documentation before requesting a PR review.
 
+
+Adding new Examples
+===================
+New application examples in ```examples``` should be commit as only ```.py``` files with code-cells. These can, for example, be used in VSCode with the python extension, or in JupyterLab with the `jupytext <https://jupytext.readthedocs.io/en/latest/install.html>`_ extension.
```
Comment (ckolbPTB) [inline docs/source/contributor_guide.rst]:
```suggestion
New exciting applications of MRpro can be added in ```examples``` as only ```.py``` files with code-cells. These can, for example, be used in VSCode with the python extension, or in JupyterLab with the `jupytext <https://jupytext.readthedocs.io/en/latest/install.html>`_ extension.
```

File: docs/source/contributor_guide.rst [hunk 2]
```diff
@@ -65,17 +65,54 @@ src/mrpro structure
...
+An automatic workflow at github will create notebooks and pages in the documentation based on the python scripts.
+The data to run the examples should be publicly available and hosted externally, for example at zenodo.
+Please be careful not to add any binary files to our commits.
```
Comment (ckolbPTB) [inline docs/source/contributor_guide.rst]:
```suggestion
Please be careful not to add any binary files to your commits.
```

File: docs/source/contributor_guide.rst [hunk 3]
```diff
@@ -65,17 +65,54 @@ src/mrpro structure
...
+
+Release Strategy
+================
+We are still in pre-release mode and do not guarantee a stable API / strict semver compatibility. We currently use ```0.YYMMDD``` as versioning and release in regular intervals to `pypi  <https://pypi.org/project/mrpro/>`_.
```
Comment (ckolbPTB) [inline docs/source/contributor_guide.rst]:
```suggestion
We are still in pre-release mode and do not guarantee a stable API / strict semantic versioning compatibility. We currently use ```0.YYMMDD``` as versioning and release in regular intervals to `pypi  <https://pypi.org/project/mrpro/>`_.
```

**Review by ckolbPTB — APPROVED** [review]

File: docs/source/contributor_guide.rst (hunk 1 above)
Comment (fzimmermann89) [reply docs/source/contributor_guide.rst]: only exciting ones :D

**Pull request merged**